### DISCUSSION TITLE
[update from 4.5.3 to 4.9.4] Strange strings show on smart albums and landing page

### DISCUSSION THREAD

**Discussion by vuongquoc** [LycheeOrg/Lychee#1941] [Q&A] [answered]
Hi all,

I've just updated lychee to 4.9.4 but...
my smart albums changed to: (lychee.UNSORTED, lychee.PUBLIC...) as shown below
https://gyazo.com/89589e23cfbd459b21f8451ec5abf3ac
same thing happened to my landing page too (lychee.GALLERY, lychee.FOOTER_Copyrigts)
https://gyazo.com/9ad71f46af8fe9f264c7e44cc837d847

How/Where should i look for fixing these?

Thank you!

**Comment by ildyria** [comment]
I don't know if that is still the case, but my instinct would tell me this is a cache problem of the JS.

**Reply by vuongquoc** [reply]
Hi sir,
It still there and i don't know where should i twitch to fix it...

Could you tell me how to?

Thank you.

**Comment by ildyria** [comment, marked as answer]
Fixed with v5.
